### PR TITLE
Update DD4L doc page

### PR DESCRIPTION
**Pull request by aiordache**
### Proposed changes

Update page for DD4L tech preview. 

Changes:
- removed several configurations from the maintainers scripts that are not needed anymore
- shared memory configuration not needed anymore - removed doc section on it
- integration with pass is complete
- dev environments are now enabled

### CODE REVIEW

File: desktop/linux/index.md [hunk 1]
```diff
@@ -24,62 +24,26 @@ To install Docker Desktop for Linux:
...
-3. Check whether the user belongs to `docker` and `kvm` groups. You may need to restart the host to load the group configuration (automated in post-install script).
 
   There are a few post-install configuration steps done through the maintainers' scripts (post-install script contained
   in the deb package.
 
   For each user, the post-install script:
 
    - installs systemd units
-   - configures `desktop-linux` as the default Docker CLI context
    - installs Compose and the `docker scan` plugins to `~/.docker/cli-plugins`
```
Comment (p1-0tr) [inline desktop/linux/index.md:37]: This is no longer done in `postinst`, and gets installed in `/usr/lib/docker/cli-plugins`

File: desktop/linux/index.md [hunk 2]
```diff
@@ -164,18 +134,14 @@ To remove Docker Desktop for Linux, run:
...
+For a complete cleanup, remove configuration and data files at `$HOME/.docker/desktop`
+```console
+$ rm -r $HOME/.docker/desktop
+```
 
- - Dev Environments are not yet available.
+## Known issues
 
- - At the end of the installation process, `apt` displays an error due to installing a downloaded package. You can ignore this error message.
+  - At the end of the installation process, `apt` displays an error due to installing a downloaded package. You can ignore this error message.
```
Comment (p1-0tr) [inline desktop/linux/index.md:144]: nit: looks like a spurious change

File: desktop/linux/index.md [hunk 3]
```diff
@@ -24,62 +24,22 @@ To install Docker Desktop for Linux:
...
-   - installs Compose and the `docker scan` plugins to `~/.docker/cli-plugins`
-   - enables Compose V2 as the default `docker-compose`
-   - adds user to `docker` and `kvm` groups
-
-  In addition, the post-install script:
+  The post-install script:
 
   - sets the capability on the Docker Desktop binary to map privileged ports and set resource limits
   - adds a DNS name for Kubernetes to `/etc/hosts`
   - creates the Docker Desktop file for the application launcher
```
Comment (p1-0tr) [inline desktop/linux/index.md:38]: Sorry, I missed that, the `*.desktop` file gets installed directly.

But, the postinstall script creates a link from `/usr/bin/docker` to `/usr/local/bin/com.docker.cli`

**Review by p1-0tr — DISMISSED** [review]

File: desktop/linux/index.md [hunk 4]
```diff
@@ -121,6 +80,13 @@ API version:       1.41
 ...
 ```
 
+> **Note:**
+>
+> Docker Desktop relies on `pass` to store credentials. Before signin in to Docker Hub from
```
Comment (usha-mandya) [inline desktop/linux/index.md:85]:
```suggestion
> Docker Desktop relies on `pass` to store credentials. Before signing in to Docker Hub from
```

File: desktop/linux/index.md [hunk 5]
```diff
@@ -164,16 +130,14 @@ To remove Docker Desktop for Linux, run:
...
-## Known issues
-
- - The Docker CLI login flow has some inconsistencies that we are currently investigating. If you experience any issues when trying to log in, remove the `credsStore` property from `~/.docker/config.json` and restart Docker Desktop (run either
- `systemctl --user restart docker-desktop` or quit Docker Desktop and relaunch).
-
- - Docker Desktop stores the passwords in base-64 encoded plaintext. Integration with `pass` is currently a work in progress.
-
- - After launching Docker Desktop, you must remove `~/.docker/scan/config.json` for `docker scan` to work.
+For a complete cleanup, remove configuration and data files at `$HOME/.docker/desktop`, the symlink at `/usr/local/bin/com.docker.cli` and purge systemd remaining service files.
+```console
```
Comment (usha-mandya) [inline desktop/linux/index.md:134]: nit: Add a blank line before the code

File: desktop/linux/index.md [hunk 6]
```diff
@@ -164,16 +130,14 @@ To remove Docker Desktop for Linux, run:
...
 
-## Known issues
-
- - The Docker CLI login flow has some inconsistencies that we are currently investigating. If you experience any issues when trying to log in, remove the `credsStore` property from `~/.docker/config.json` and restart Docker Desktop (run either
- `systemctl --user restart docker-desktop` or quit Docker Desktop and relaunch).
-
- - Docker Desktop stores the passwords in base-64 encoded plaintext. Integration with `pass` is currently a work in progress.
-
- - After launching Docker Desktop, you must remove `~/.docker/scan/config.json` for `docker scan` to work.
+For a complete cleanup, remove configuration and data files at `$HOME/.docker/desktop`, the symlink at `/usr/local/bin/com.docker.cli` and purge systemd remaining service files.
```
Comment (usha-mandya) [inline desktop/linux/index.md:133]: Should this be '..., and purge the remaining systemd service files'?

**Pull request merged**